src/commcare_cloud/ansible/roles/citusdb/tasks/main.yml

```diff
@@ -20,6 +20,10 @@
    apt: 
     name: "{{citus_binary}}"
     state: present
+
+ - name: Add citus to postgres prelod libraries
+   shell: "pg_conftool 11 main set shared_preload_libraries 'citus, pg_stat_statements'"
```

Comment:
this should use `{{postgresql_version}}`

Reply:
Updated